packages/multichain-network-controller/src/utils.ts

```diff
@@ -33,3 +37,43 @@ export function checkIfSupportedCaipChainId(
   // Check if the chain id is supported
   return Object.keys(AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS).includes(id);
 }
+
+/**
+ * Updates a network configuration to the format used by the MultichainNetworkController.
+ * This method is exclusive for EVM networks with hex identifiers from the NetworkController.
+ *
+ * @param network - The network configuration to update.
+ * @returns The updated network configuration.
+ */
+export const updateNetworkConfiguration = (
+  network: NetworkConfiguration,
+): MultichainNetworkConfiguration => {
+  return {
+    chainId: toCaipChainId('eip155', hexToNumber(network.chainId).toString()),
+    isEvm: true,
+    name: network.name,
+    // @ts-expect-error - The nativeCurrency is not typed in the NetworkController
+    nativeCurrency: network.nativeCurrency,
```

Comment:
See my other comment about `string` vs `CaipAssetType`, but the mapping should probably be done here.

Reply:
Based on our conversation. https://github.com/MetaMask/core/commit/11d4258c4f8be688bfc3d54757b4735ebf6b97b2